bot/__main__.py

```diff
@@ -12,8 +12,8 @@
 
 async def main_async() -> None:
     secrets = load_secrets()
-    translations = load_translations()
     data = load_data()
+    translations = load_translations(data)
```

Comment:
But we don't need to pass all of it in, just the part the function is concerned about, `disabled_translations: list[str]`